parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -640,7 +640,7 @@ configGPUDrivers() {
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         mkdir -p /opt/{actions,gpu}
         if [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
-            ctr image pull $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG
+            ctr image pull $NVIDIA_DRIVER_IMAGE:$NVIDIA_DRIVER_IMAGE_TAG > /dev/null
```

Comment:
are we sure we want to silence stdout here?

Reply:
The output from ctr image pull is really noisy - without these changes the output of the build step is 250k lines, with them it's only 20k. The volume was making it really hard to read the logs. And they're very much meant for interactive usage - showing each layer's download progress, and I think it's using ^r characters to update them (which doesn't work in this context). Any problems still get reported on stderr.

Reply:
fair enough, and I don't think we ever really did much debugging using the output from these commands anyways